src/filters/filters.ml

```diff
@@ -732,7 +732,6 @@ let run tctx main before_destruction =
 		"add_final_return",if com.config.pf_add_final_return then add_final_return else (fun e -> e);
 		"RenameVars",(match com.platform with
 		| Eval -> (fun e -> e)
-		| Jvm -> (fun e -> e)
```

Comment:
Note to self: While the JVM target generally doesn't care about variable names, there is one exception: captured locals might become fields in the generated closure classes, and those are supposed to be unique. This has never come up before, but the newly added tests caused a case where this was failing.